sdk/highlight-ex/lib/highlight.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Highlight do
+  @moduledoc """
+  Documentation for Highlight.
+  """
+  require OpenTelemetry.Tracer, as: Tracer
+  require Logger
+
+  defmodule Config do
+    @enforce_keys [:project_id]
+    defstruct [
+      :project_id,
+      :service_name,
+      :service_version
+    ]
+  end
+
+  @doc """
+  Initialize the Highlight SDK with the given configuration.
+  This sets up Elixir for automatic log collection.
+
+  ## Examples
+
+    iex> Highlight.init()
+  """
+  def init() do
```

Comment:
could we add docs on how to set up the SDK, ie. https://github.com/highlight/highlight/blob/64827f23e091bed85e79756e5370745a26337b12/highlight.io/components/QuickstartContent/traces/dotnet/dot-net.tsx#L30